pkg/netlinkwrapper/netlink.go

```diff
@@ -103,15 +126,27 @@ func (*netLink) LinkSetUp(link netlink.Link) error {
 }
 
 func (*netLink) LinkList() ([]netlink.Link, error) {
-	return netlink.LinkList()
+	var links []netlink.Link
+	var err error
+	err = retryOnErrDumpInterrupted(func() error {
+		links, err = netlink.LinkList()
+		return err
+	})
+	return links, discardErrDumpInterrupted(err)
```

Comment:
What is the rationale behind discarding this error type? If we retried already and it still occurred, user would need to know about this error right? I know we are printing log but user would not see that. Also if that error occurs are links stale in that case or empty?